components/InputText.js

```diff
@@ -10,7 +10,7 @@ import {
   Platform
 } from 'react-native'
 import Icon from 'react-native-vector-icons/Ionicons'
-import styles, {mainThemeColor} from '../styles'
+import styles, { mainThemeColor } from '../styles'
```

Comment:
this file's content doesn't seem to be changed except the spacing. Can you revert this file as well?